schema.graphql

```diff
@@ -211,6 +211,15 @@ input DonationWhereInput {
   name_of_user_starts_with: String
 }
 
+input EditVenueInput {
+  _id: ID!
+  capacity: Int
+  description: String
+  file: String
+  name: String
+  organizationId: ID!
```

Comment:
No need for `organizationId` here.

Reply:
`organizationId` is being used here to check here if the user who wants to edit the venue is admin of the organization or not. Should I take any other approach?

Reply:
Fetch it from the database. The id is a unique identifier in itself. We can't let the user provide a faulty organizationId or change the organization itself in the venue.